app/policies/upcoming/read/facility_group_policy.rb

```diff
@@ -0,0 +1,15 @@
+class Upcoming::Read::FacilityGroupPolicy < Upcoming::ApplicationPolicy
```

Comment:
These are currently namespaced under `Manage` or `Read`, but they can be totally collapsed to be 1 policy if we just pass in read or write from the call-site as necessary.